lib/ace/ext/papeeria/tex_completer.coffee

```diff
@@ -75,48 +82,85 @@ define( (require, exports, module) ->
   equationKeywords = ['\\alpha']
   listKeywords = ['\\item']
 
-  listKeywords_ = listKeywords.map((word) ->
+  listKeywords = listKeywords.map((word) ->
     caption: word,
     value: word
     meta: 'list'
   )
-  equationKeywords_ = equationKeywords.map((word) ->
+  equationKeywords = equationKeywords.map((word) ->
     caption: word,
     value: word
     meta: 'equation'
   )
-  
+
+  # Specific for token's system of type in ace
+  # We saw such a realization in html_completions.js
+  isType = (token, type) ->
+    return token.type.lastIndexOf(type) > -1
```

Comment:
just curious: if `token.type == "qweqweqFOOOasdfasdfasdf"` then `isType(token, "FOO")` is also true. Is it expected? Is my understanding correct that instead of something like `"lparen.ref".split(".").indexOf("ref")` we do `"lparen.ref".lastIndexOf("ref") > -1` hoping that (fingers crossed) we'll never meet `"lparen.reference"`, which returns `false` in the former case and `true` in the latter?